_posts/2016-06-20-user-testing.md

```diff
@@ -0,0 +1,149 @@
+
+- What are other sites doing well? What improvements could be applied to our client's site?
+- Where are other sites falling short? How can we ensure these bases are covered on our client's site?
+- What user experience problems exist on other sites? Can we test for these same issues on our client's site during the user testing phase?
+- How does our client's site compare overall in terms of quality? How can we elevate our client's site among its peers?
+
+### User testing
+
+After gathering information from analytics and competitive analysis, we design our user tests. These are in-person meetings with a single participant at a time who is asked to complete tasks on the website, acting as one of the user personas. After reading the persona description and the task to the user, we observe their experience and record how they interact with the site.
+
+##### Designing the tests
+
+Designing the user tests is a multi-step process:
+
+1. Decide which personas will be tested and how many participants are needed. User personas to be tested are chosen based on the client's priority and potential/known issues found in the review of analytics and competitors. Number of participants can be surprisingly small and still quite effective, as per the Nielson Group's [detailed analysis](https://www.nngroup.com/articles/why-you-only-need-to-test-with-5-users/). Ideally user testing would be iterative -- a few users would be tested, followed by improvements to the site targeting the results of those tests, followed by more testing, etc. If this isn't possible, five participants with one round of testing is typically effective.
+2. Determine what needs to be tested. These items come from the critical tasks for our user persona(s), the potential points of frustration or difficulty for users (found by Google analytics, competitive analysis, or simply review of the current site), and anything else desired by the client. I frame these items as goals.
+3. Translate goals to tasks. At this stage, each goal is translated into an actionable task or endpoint for the user.
+4. Translate tasks to scenarios. Rarely is a user told exactly what to do on a website in real life. To make the tests more realistic, we translate tasks into scenarios. We'll read the scenario to each user to see if she can complete the desired task, how she approaches the task, and what points of frustration she encounters along the way.
+
+For example:
+
+**Goal:** The newsletter signup should be well-placed and easy to use.
+
+**Task:** Sign up for the newsletter.
+
+**Scenario:** You want to keep up to date with news and job opportunities at XYZ Company. Using the website, find a way to receive monthly email updates from XYZ Company.
+
+[This article](https://www.nngroup.com/articles/task-scenarios-usability-testing/) provides detailed recommendations for creating effective scenarios.
+
+Some other tips for designing user tests:
```

Comment:
We did this recently, but we recruited people who were intentionally not familiar with the site. Is that useful to put in as a recommendation or tip?